From 1df9f4b48b3239e2cc4ccfdc784c037da9da4c74 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <heinrich.schuchardt@canonical.com>
Date: Tue, 23 Nov 2021 18:25:35 +0100
Subject: [PATCH 2/3] riscv: add clint,has-no-64bit-mmio to Nehza DT

Upstream OpenSBI only supports clint,has-no-64bit-mmio to indicate that the
clint has only 32bit mmio.

Signed-off-by: Heinrich Schuchardt <heinrich.schuchardt@canonical.com>
---
 arch/riscv/dts/sun20i-d1.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/riscv/dts/sun20i-d1.dtsi b/arch/riscv/dts/sun20i-d1.dtsi
index 56d02c33b0..1bc5ae762e 100644
--- a/arch/riscv/dts/sun20i-d1.dtsi
+++ b/arch/riscv/dts/sun20i-d1.dtsi
@@ -1151,6 +1151,7 @@
 				     "sifive,clint0";
 			reg = <0x14000000 0xc000>;
 			reg-io-width = <4>;
+			clint,has-no-64bit-mmio;
 			interrupts-extended = <&cpu0_intc 3>,
 					      <&cpu0_intc 7>;
 		};
-- 
2.32.0

